Gems/PhysX/Code/Editor/KinematicDescriptionDialog.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "KinematicDescriptionDialog.h"
+
+AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
+#include <Editor/ui_KinematicDescriptionDialog.h>
+AZ_POP_DISABLE_DLL_EXPORT_MEMBER_WARNING
+
+static constexpr AZStd::string_view dynamicDescription = "With <b>Dynamic</b> rigid bodies, you can physics forces "
+                                                  "(gravity, collision, etc.) to control the movement and "
+                                                  "position of the object.";
+
+static constexpr AZStd::string_view kinematicDescription = "With <b>Kinematic</b> rigid bodies, you can use Transform to "
+                                                    "control the movement and position of the object. ";
+
+static constexpr AZStd::string_view moveWithCode = "Move with code";
+static constexpr AZStd::string_view moveManually = "Move manually";
+static constexpr AZStd::string_view collisions = "Collisions";
+static constexpr AZStd::string_view gravity = "Gravity";
```

Comment:
I'd recommend to add `Text` at the end of the variables' name so it's clear what the variable contains, for example `gravity` -> `gravityText`. Same with `moveWithCode`, `moveManually` and `collisions`.

Reply:
or `Description` or `Name` works too, not sure what we use most often but being a little more explicit is definitely a good idea